packages/console-client/.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extends: ["../../.eslintrc.cjs"],
```

Comment:
the mia linter will be used here? 
I don't know if the intent is to use it somewhere...

Reply:
the intent is to extend the base eslint config with the package specific one

i need this as all the files under `kiota-client` directory are violating our eslint conf (these files are autogenerated, thus we don't want to fix em every time the client is generated)